drinks/src/main/java/ca/hendriks/bartender/drinks/ingredient/Ingredient.java

```diff
@@ -14,16 +14,20 @@ public class Ingredient {
 
     @Column
     @Enumerated(EnumType.STRING)
-    private IngredientType ingredientType;
+    private IngredientType type;
 
     public Ingredient() {
         // JSR 338: The entity class must have a no-arg constructor.
     }
 
-    public Ingredient(final Integer id, final String name, final IngredientType ingredientType) {
+    public Ingredient(final String name, final IngredientType type) {
```

Comment:
So the name will be the PK?

Reply:
The k is the Internet ID. It's set by Hibernate on save.